Client/YooChooseNotifier.php

```diff
@@ -175,4 +177,62 @@ private function getNotificationEndpoint()
             $this->options['customer-id']
         );
     }
+
+    /**
+     * Returns the YooChoose recommendation endpoint
+     *
+     * @return string
+     */
+    private function getRecommendationEndpoint()
+    {
+        return sprintf(
+           '%s/api/%s',
+            rtrim( $this->options['reco-api-endpoint'], '/' ),
+            $this->options['customer-id']
+        );
+    }
+
+    /**
+     * Fetch recommendations from YooChoose
+     *
+     * @param $userId
```

Comment:
Please add types to the phpdoc. No description required, parameters are pretty straightforward.